### DISCUSSION TITLE
addValidationError

### DISCUSSION THREAD

**Discussion by pahaz** [keystonejs/keystone#5881] [Feature requests]
I have a simple use case. I want to display field errors near the field. The best way for that is to add a `fieldPath` in error data context.

Example `argumentName ` from Apollo doc: https://www.apollographql.com/docs/apollo-server/data/errors/#including-custom-error-details

At the moment there is no way to do it inside the admin UI.

Could you please add it in the validation error context?

**Comment by bladey** [comment]
Thanks for the suggestion @pahaz!
